### PR TITLE
Adding user creation form

### LINKED ISSUE #101: Add user creations form

### PR DESCRIPTION
**Pull request by Micha546**
This form is not to be used alone, only with Apartment creation form
and Seeker creation form which will come in a later PR
Added tests to test the form.
Fix #101

### CODE REVIEW

File: users/forms.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        if not password2:
+            raise forms.ValidationError('You must confirm your password')
+        if password1 != password2:
+            raise forms.ValidationError('Your passwords do not match')
+        return password2
+
+    def save(self, commit=True):
+        new_user = super(UserCreationForm, self).save(commit=False)
+        new_user.set_password(self.cleaned_data["password1"])
+        if commit:
```
Comment (DanielMy777) [inline users/forms.py:34]: What's this 'commit'?

Comment (Micha546) [reply users/forms.py:34]: If its True then the function saves the User to the database, otherwise (commit=False) it doesn't

**Review by DanielMy777 — COMMENTED** [review]
Looks good!

File: users/forms.py (hunk 1 above)
Comment (simzacks) [reply users/forms.py:34]: Why would you call save if you didn't want to save it?

Comment (Micha546) [reply users/forms.py:34]: When you use save with commit=False it returns an instance of the model the form is attached to. It just doesn't save it to the data base right away. So if you want to modify the instance more after the form is done, you can

**Comment by AmitAharoni** [comment]
Good job :)

**Review by simzacks — APPROVED** [review]

**Review by OrAsaf — APPROVED** [review]

**Pull request merged**